environments/demo/demo.tfvars

```diff
@@ -20,5 +20,26 @@ frontends = [
     health_path         = "/"
     health_protocol     = "Https"
     cache_enabled       = "false"
+  },
```

Comment:
you don't need front door in demo , it is a special case.

Reply:
We do in this case, because we are sending to a different backend.